CMakeLists.txt

```diff
@@ -728,6 +728,14 @@ if(BUILD_TESTS)
     )
   endif()
 
+  add_e2e_test(
+    NAME e2e_localcommithandler
+    PYTHON_SCRIPT ${CMAKE_SOURCE_DIR}/tests/e2e_localcommithandler.py
+    CONSENSUS cft
+    CONTAINER_NODES TRUE
+    ADDITIONAL_ARGS -p "samples/apps/localcommithandler/liblocalcommithandler"
+  )
```

Comment:
Is there a specific reason why this is a separate app/end-to-end test rather than fold it in the existing logging C++ app that demonstrates most (all?) C++ API calls?

Reply:
No specific reason besides that I wasn't wanting to clog up the logging demo but happy to merge it if that's still desired.

Reply:
My understanding is that the logging app is already clogged up as it demonstrates the entire C++ API rather than a minimal getting started sample. @eddyashton, what do you think?

Reply:
Yeah let's roll this into the logging sample app for now.

Reply:
Had a go at adding a new endpoint.